### PR TITLE
Improved the TFX OSS notebook and README

### PR DESCRIPTION
**Pull request by Ark-kun**
Addressed the issues described here: https://github.com/kubeflow/pipelines/pull/913

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/922)

### CODE REVIEW

File: samples/tfx-oss/README.md [hunk 1]
```diff
@@ -39,9 +47,9 @@ Modify the pipeline configuration file at
 tfx/examples/chicago_taxi_pipeline/taxi_pipeline_kubeflow_large.py
 ```
 Configure 
-- GCS storage bucket name (replace "my-bucket")
-- GCP project ID (replace "my-gcp-project")
-- Make sure the path to the taxi_utils.py is correct
+- Set `_input_bucket` to the GCS directory where you've copied taxi_utils.py. I.e. gs://<my bucket>/<path>/
+- Set `_output_bucket` to the GCS directory where you've want the results to be written
+- Set GCP project ID (replace my-gcp-project). Note that it should be project ID (usually has numbers in the end), not project name.
```
Comment (neuromage) [inline samples/tfx-oss/README.md]: The numbers in the end bit isn't true, so I won't point that out. GCP project id is sufficient. Drop the it from 'Note...' ?

File: samples/tfx-oss/README.md [hunk 2]
```diff
@@ -9,6 +9,8 @@ This pipeline demonstrates the TFX capablities at scale. The pipeline uses a pub
 
 ## Setup
 
+Enable DataFlow API for your GKE cluster: <https://console.developers.google.com/apis/api/dataflow.googleapis.com/overview>
```
Comment (neuromage) [inline samples/tfx-oss/README.md]: @IronPan when we deploy a kubeflow cluster through kubeflow deploy, don't we get these cloud apis enabled for us? Or do we need to be explicit about it?

File: samples/tfx-oss/TFX Example.ipynb [hunk 3]
```diff
@@ -19,6 +19,14 @@
     "!pip3 install https://storage.googleapis.com/ml-pipeline/release/0.1.10/kfp.tar.gz --upgrade\n"
    ]
   },
```
Comment (paveldournov) [inline samples/tfx-oss/TFX Example.ipynb]: Can you also add a note that this notebook should be run in the KFP cluster?

File: samples/tfx-oss/README.md (hunk 2 above)
Comment (Ark-kun) [reply samples/tfx-oss/README.md]: BTW, I did not need to explicitly enable BigQuery.

File: samples/tfx-oss/README.md (hunk 1 above)
Comment (Ark-kun) [reply samples/tfx-oss/README.md]: Done.

File: samples/tfx-oss/TFX Example.ipynb (hunk 3 above)
Comment (Ark-kun) [reply samples/tfx-oss/TFX Example.ipynb]: Done.

**Comment by cjwagner** [comment]
/hold

Holding this PR because it has a non-successful GitHub check which is preventing Tide from merging because Tide does not respect GitHub checks (only GitHub status contexts). This is blocking automated merge to this repo.

**Comment by Ark-kun** [comment]
> Holding this PR because it has a non-successful GitHub check which is preventing Tide from merging 
@cjwagner 
Can you point me to any unsuccessful check?

**Comment by cjwagner** [comment]
> Can you point me to any unsuccessful check?

The check was in the pending state when I wrote that message. It appears to be successful now so Tide might be able to merge now.  That being said Tide doesn't respect GH checks so the behavior is undefined and the checks really should not be enabled on this repo at all. I opened an issue for that: https://github.com/kubeflow/pipelines/issues/930

**Pull request merged**